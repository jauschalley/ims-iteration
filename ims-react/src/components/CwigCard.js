import React from 'react';
import {Container} from 'react-bootstrap';

const CwigCard = ({children}) => {
  return(
    <Container className="cwig-card">
      {children}
    </Container>
  );
};

export default CwigCard;