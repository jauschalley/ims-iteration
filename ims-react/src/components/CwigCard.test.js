import React from 'react';
import renderer from 'react-test-renderer';
import CwigCard from './CwigCard';

describe('<CwigCard />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <CwigCard />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});