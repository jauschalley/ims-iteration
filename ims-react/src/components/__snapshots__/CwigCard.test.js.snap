// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CwigCard /> renders correctly 1`] = `
<div
  className="cwig-card container"
/>
`;
